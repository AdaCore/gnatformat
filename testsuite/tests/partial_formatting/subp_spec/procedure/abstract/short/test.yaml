description: SubpSpec formatting with the selection of an abstract procedure.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 4",
    "-SC 5",
    "-EL 4",
    "-EC 10",
  ]
status_code: 1