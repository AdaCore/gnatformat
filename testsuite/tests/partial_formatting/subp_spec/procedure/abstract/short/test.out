<CompilationUnit test.ads:1:1-10:10>
test.ads(<SubpSpec test.ads:4:4-4:43>) - 4:1-4:43
^
   procedure Bar (Self : Foo; I : Integer)$

